<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsDialog</class>
 <widget class="QDialog" name="ItemsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of items: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_num_of_items">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Location in repository:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="locationDirRelPath">
       <property name="toolTip">
        <string>Path where to put the files in the repository.</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSelectLocationDirRelPath">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_fields">
       <item>
        <widget class="QLabel" name="label_fields">
         <property name="text">
          <string>Fields:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_tags">
       <item>
        <widget class="QLabel" name="label_tags">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_fields_add">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Fields to add:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_tags_add">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Tags to add:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_fields_rm">
       <item>
        <widget class="QLabel" name="label_fields_rm">
         <property name="text">
          <string>Field names to remove:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_tags_rm">
       <item>
        <widget class="QLabel" name="label_tags_rm">
         <property name="text">
          <string>Tags to remove:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
